inspirehep/dojson/common/base.py

```diff
@@ -216,10 +217,13 @@ def collections(self, key, value):
 
     def get_value(value):
```

Comment:
Plus, for a file this big I find it easier to read if the rules are actually self contained.